config.go

```diff
@@ -50,20 +50,31 @@ type CachingConfig struct {
 	// CacheType represents the type of cache that we wish to use: "memory", "filesystem", or "redis"
 	CacheType     string                `toml:"cache_type"`
 	RecordTTLSecs int64                 `toml:"record_ttl_secs"`
-	Redis         RedisConfig           `toml:"redis"`
+	Redis         RedisCacheConfig      `toml:"redis"`
 	Filesystem    FilesystemCacheConfig `toml:"filesystem"`
 	ReapSleepMS   int64                 `toml:"reap_sleep_ms"`
 	Compression   bool                  `toml:"compression"`
+	BoltDB        BoltDBCacheConfig     `toml:"boltdb"`
 }
 
-// RedisConfig is a collection of Configurations for Connecting to Redis
-type RedisConfig struct {
+// RedisCacheConfig is a collection of Configurations for Connecting to Redis
+type RedisCacheConfig struct {
 	// Protocol represents the connection method (e.g., "tcp", "unix", etc.)
 	Protocol string `toml:"protocol"`
 	// Endpoint represents FQDN:port or IPAddress:Port of the Redis server
 	Endpoint string `toml:"endpoint"`
 }
 
+// BotlDbCacheConfig is a collection of Configurations for storing cached data on the Filesystem
```

Comment:
BotlDb :)

Reply:
Damn you!!! :-)     No wonder my `sed` didn't pick that one up